lib/foreman/renderer/scope/macros/snippet_rendering.rb

```diff
@@ -7,15 +7,6 @@ def snippet_if_exists(name, options = {})
             snippet(name, { silent: true }, variables: options[:variables])
           end
 
-          # provide embedded snippets support as simple erb templates
-          def snippets(file, options = {})
-            if source.find_snippet(file)
-              snippet(file.gsub(/^_/, ""), options)
```

Comment:
If i understand correctly, this branch is not broken, so it would be best to deprecate it before removing